scan-command/src/main/java/io/ballerina/scan/utils/ScanUtils.java

```diff
@@ -512,26 +526,28 @@ private static Consumer<TomlValueNode> loadRules(ScanTomlFile scanTomlFile, bool
     }
 
     /**
-     * Downloads the remote JAR file to the target directory.
+     * Returns the {@link Optional<String>} path of the downloaded remote JAR file.
      *
      * @param targetDir     the target directory
      * @param fileName      the remote file name
      * @param remoteJarFile the remote JAR file path
      * @param outputStream  the print stream
-     * @return the path of the downloaded remote JAR file
+     * @return the optional path of the downloaded remote JAR file
      */
-    private static String loadRemoteJAR(Path targetDir, String fileName, URL remoteJarFile, PrintStream outputStream) {
+    private static Optional<String> loadRemoteJAR(Path targetDir, String fileName, URL remoteJarFile,
+                                                  PrintStream outputStream) {
         Path cachedJarPath = targetDir.resolve(fileName + JAR_PREDICATE);
         if (Files.exists(cachedJarPath)) {
             outputStream.println("Loading " + fileName + JAR_PREDICATE + " from cache...");
```

Comment:
Let's make all the logs similar in format. Here you have used `...`, perhaps to show that it's a continuing process. However in L546, the message is `Downloading remote JAR: xxxxx`, with no dots appended. IMO we should simply display output logs with no exclamations, dot sequences appended to the end.